database/engine/metadata_log/logfile.c

```diff
@@ -667,7 +667,7 @@ static int scan_metalog_files(struct metalog_instance *ctx)
     metalog_parser_object.enabled = cd.enabled;
     metalog_parser_object.host = ctx->rrdeng_ctx->host;
     metalog_parser_object.cd = &cd;
-    metalog_parser_object.trust_durations = 0;
+    metalog_parser_object.usec_semantics = PLUGINSD_USEC_IGNORE;
     metalog_parser_object.private = &metalog_parser_state;
 
     PARSER *parser = parser_init(metalog_parser_object.host, &metalog_parser_object, NULL, PARSER_INPUT_SPLIT);
```

Comment:
We are writing an error message when `parser_init` fails, but `parser_init` uses `callocz` to allocate, this means that if a problem happens, it will call `fatal` and shutdown Netdata completely. We need to think about these messages we are writing after to allocate the `PARSER`.

Reply:
If `callocz` fails then the machine is out of memory and shutting down Netdata seems like a good choice. What problem do you have in mind?

Reply:
I thought you wanted to give a more specific information that is missed when callocz fails, this was the motive I called attention for this. When `callocz` fails the information is more generic.